core/src/main/scala/com/microsoft/azure/synapse/ml/fabric/OpenAITokenLibrary.scala

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.azure.synapse.ml.fabric
+
+import com.microsoft.azure.synapse.ml.logging.SynapseMLLogging
+import spray.json.DefaultJsonProtocol.StringJsonFormat
+
+object OpenAITokenLibrary extends SynapseMLLogging with AuthHeaderProvider {
+  var MLMWCToken = "";
```

Comment:
can we try to eliminate this var by making it a def or something?

Reply:
I need another var to tell it is mwc token or aad token and I think it is strange to change value of another var in def for this var. And also this var is a catched token and will expire sometime while def is called only once, wdyt?